programs/psinode/main.cpp

```diff
@@ -2460,6 +2584,8 @@ int main(int argc, char* argv[])
    // specify default token/service
    opt("leeway", po::value<uint32_t>(&leeway_us)->default_value(200000),
        "Transaction leeway, in µs.");
+   opt("http-timeout", po::value(&http_timeout)->default_value({}, "")->value_name("seconds"),
+       "The maximum time for HTTP clients to send or receive a message");
```

Comment:
I think something like "The timeout before the server closes idle HTTP connections" is a clearer description.